docs/install-nuget-client-tools.md

```diff
@@ -40,7 +40,7 @@ Package Manager Console commands work only within Visual Studio on Windows and d
 
 ## Support policy
 
-The Visual Studio for Windows support policy can be found at [Visual Studio Product Lifecycle and Servicing](https://learn.microsoft.com/visualstudio/productinfo/vs-servicing).
+The Visual Studio for Windows support policy can be found at [Visual Studio Product Lifecycle and Servicing](https://aka.ms/vsservicing).
```

Comment:
```suggestion
The Visual Studio for Windows support policy can be found at [Visual Studio Product Lifecycle and Servicing](/visualstudio/productinfo/vs-servicing).
```

The PR title and description claim it's changing to a relative link, but it's not. It's using an absolute link to the aka.ms short link service.

I don't know how common it is, but if a customer changes languages from what learn.microsoft.com defaults to, then clicking the aka.ms link will show them the docs in the auto-detected language, not the language they manually switched to.